src/platform/user/authorization/components/RequiredLoginView.jsx

```diff
@@ -161,9 +164,14 @@ RequiredLoginView.propTypes = {
     PropTypes.arrayOf(validService),
   ]).isRequired,
   user: PropTypes.object.isRequired,
+  useSiS: PropTypes.bool,
   verify: PropTypes.bool,
 };
 
-export default RequiredLoginView;
+const mapStateToProps = state => {
+  return { useSiS: state.featureToggles.signInServiceEnabled || true };
+};
+
+export default connect(mapStateToProps)(RequiredLoginView);
```

Comment:
Out of curiosity, if there a reason you prefer using `connect` over the Redux hooks like useSelector?